plugins/inputs/kube_inventory/kube_inventory.go

```diff
@@ -67,6 +70,14 @@ func (ki *KubernetesInventory) Init() error {
 	if err != nil {
 		return err
 	}
+	if ki.ResponseTimeout < config.Duration(time.Second) {
+		ki.ResponseTimeout = config.Duration(time.Second * 5)
+	}
```

Comment:
This change was made to align with what the docs say is the default of 5 seconds? Are you avoiding the zero value in case someone has actually turned off the timeout?

Reply:
good idea